app/src/main/java/com/animal_villa/dto/Character.kt

```diff
@@ -1,4 +1,11 @@
 package com.animal_villa.dto
+import androidx.room.Entity
 
+@Entity
 class Character {
+    lateinit var name:String
+    var age:Int=0;
+    lateinit var job:String
+    lateinit var statusValues:MutableList<Int> //TODO dont know what a status value is so I made it an int
+
 }
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.